tests/test_graphical_units/test_threed.py

```diff
@@ -34,6 +34,16 @@ def construct(self):
         self.play(Animation(cube))
 
 
+class FixedInFrameMObjectTest(ThreeDScene):
+    def construct(self):
+        axes = ThreeDAxes()
+        self.set_camera_orientation(phi=75 * DEGREES, theta=-45 * DEGREES)
+        text3d = TextMobject("This is a 3D text")
```

Comment:
`TextMobject` has been deprecated; please use `Text`. 

Without specifying the font to use, there is no chance this test will pass on all of our supported operating systems: different systems use different default fonts, this is my guess why the tests are failing.

Reply:
Good catch!
Thanks for this!
Therefore, I think it would be best if you could just update your test, @BrendanMartin.

Reply:
Actually, this is wrong as has been pointed out on Discord: `TextMobject` was deprecated, but it is the old equivalent to what is `Tex` nowadays. I have no idea why the test outputs are different in that case, unfortunately.